python/llm/setup.py

```diff
@@ -128,6 +129,10 @@ def setup_package():
 
     for url in lib_urls[platform_name]:
         download_libs(url, change_permission=change_permission)
+    
+    install_requires=[]
+    if "--all" in sys.argv:
+        install_requires += CONVERT_DEP
```

Comment:
let's use `extra_requires` in metadata for this

Reply:
resolved